lib/Controller/Display.php

```diff
@@ -1025,6 +1047,44 @@ function edit($displayId)
         $display->timeZone = $this->getSanitizer()->getString('timeZone');
         $display->displayProfileId = $this->getSanitizer()->getInt('displayProfileId');
 
+        $currentProfile = $display->getSettings();
```

Comment:
Not clear why we copy all current settings into the override config here? shouldn't we only do the ones that are set (which at the moment can only be the versionMediaId) ?